docs/plone_training_config/instructions_plone5.md

```diff
@@ -78,19 +78,19 @@ mkdir training
 cd training
 git clone https://github.com/collective/training_buildout.git buildout
 cd buildout
-python3.7 -m venv .
+python3.9 -m venv .
 ./bin/pip install -r requirements.txt
 ```
 
-This creates a virtualenv with Python 3.7 in the folder {file}`buildout` and installs some requirements in it.
+This creates a virtualenv with Python 3.9 in the folder {file}`buildout` and installs some requirements in it.
 
 Now you can run the buildout for the first time:
 
 ```shell
 ./bin/buildout
 ```
 
-This will take **very long** time and produce a lot of output because it downloads and configures more than 260 Python packages. Once it is done you can start your Plone instance with
+This will take **a long time** (~10 minutes on the least powerful Linode) and will produce a lot of output because it downloads and configures more than 260 Python packages. Once it is done you can start your Plone instance with
```

Comment:
```suggestion
This will take **a long time** (~10 minutes on the least powerful Linode) and will produce a lot of output because it downloads and configures more than 260 Python packages.
Once it is done, you can start your Plone instance with the following command.
```